decoders/connector/milesight/ws202/v1.0.0/payload-config.jsonc

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "../../../../../schema/connector_details.json",
+  "description": "../description.md",
+  "install_text": "**Milesight WS202** is a LoRa PIR sensor based on passive infrared technology to detect motion or occupancy. The WS202 can detect whether there is movement within a range of 6-8 meters and send the changes via the LoRaWAN® network. Additionally, the WS202 is equipped with a light sensor that can link PIR detection results to trigger scenes. With its compact size and battery power, the WS202 is easy to install anywhere. \n\nCompliant with the Milesight LoRaWAN® gateway and Milesight IoT Cloud solution, users can monitor alarms or room occupancy and easily trigger other sensors or appliances remotely via a webpage or mobile app.\n\nThe WS202 can be widely used in smart homes, smart offices, schools, warehouses, and more.\n\n**Features**\n* Occupancy or motion detection based on passive infrared and Fresnel Lens\n* Easily to install with the convenient size\n* Built-in light sensor, combine PIR sensor to achieve triggers\n* Ultra-wide-distance wireless transmission up to line of sight of 15 km\n* Equipped with NFC for one touch configuration, support card emulation mode\n* Support Milesight LoRa D2D protocol to enable ultra-low latency and directly control without gateway\n* Function well with standard LoRaWAN® gateways and network servers\n* Compliant with Milesight IoT Cloud",
```

Comment:
Remove subjective information: "users can monitor alarms or room occupancy and **_easily_** trigger other sensors or appliances remotely via a webpage or mobile app."